timemachine/cpp/src/kernels/k_nonbonded.cuh

```diff
@@ -279,9 +275,7 @@ void __device__ v_nonbonded_unified(
 
             if (COMPUTE_U) {
                 // If the energy is overflowed, don't bother adding it to the buffer
```

Comment:
Is this comment still relevant?

Reply:
Removed comment in 894b0b62bcf99a2891b80a4414e5150e0f67a5bc